app/views/pages/about.html.erb

```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
```

Comment:
This whole page is still rendered within the yield tag of the application html so these doctype, head, and body tags aren't needed.